examples/docker-compose/docker-compose-standalone-nlu.yaml

```diff
@@ -69,13 +68,14 @@ services:
       - pgdata:/var/lib/postgresql/data
 
   nginx:
-    container_name: nginx
-    build: ./nginx
+    image: nginx:1.17
```

Comment:
I think you forgot to bump this one too?

Reply:
True, I only check the NGINX folder